takeshape/penny
# Add Shopify customer sign in and custom pages

### Test Plan (Steps to test):

1. Create an account from the link at the top
1. Sign into that account
1. Sign out from the log out link on the account page
2. Sign in again
3. Add to your cart, checkout. You should see your email.
